app/src/main/java/org/mozilla/focus/activity/OnboardingActivity.kt

```diff
@@ -0,0 +1,48 @@
+package org.mozilla.focus.activity
+
+import android.os.Bundle
+import android.preference.PreferenceManager
+import android.support.v7.app.AppCompatActivity
+import org.mozilla.focus.R
+
+import kotlinx.android.synthetic.main.activity_onboarding.*
+import kotlinx.android.synthetic.main.content_onboarding.*
+import org.mozilla.focus.webview.TrackingProtectionWebViewClient
+
+class OnboardingActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_onboarding)
+
+        enable_turbo_mode.setOnClickListener { _ ->
+            setTurboMode(true)
+            setOnboardShown()
+            finish()
+        }
+
```

Comment:
This needs one more line to set the default state:
 `enable_turbo_mode.isPressed = true`

Reply:
This seems to work fine without the suggested change, both my device and liuche's.